### ISSUE TITLE
cannot build server solution

### ISSUE THREAD

**Issue by 0xLuca** [citizenfx/fivem#1163]
I followed the steps as stated at https://github.com/citizenfx/fivem/blob/master/docs/building.md, 
but when trying to build the server solution in visual studio i get these errors:

`MSB8066 Custom build for '..\..\..\vendor\node_inspector.py' exited with code 1.`
`LNK1181 cannot open input file 'C:\Users\luczi\projects\fivem\server\code\build\server\windows\obj\Debug\node\lib\node.lib'`

The file does not exist, neither its parent folder "lib":
![grafik](https://user-images.githubusercontent.com/59053718/143896950-a25b84b7-b5f1-4c17-80ee-6f19c63e3a9c.png)

**Comment by blattersturm** [comment]
Are you using Python 3.10 by chance?

**Comment by 0xLuca** [comment]
> Are you using Python 3.10 by chance?

Yes, i am using 3.10.0 from the official repository

**Comment by blattersturm** [comment]
Right, someone claimed this was incompatible before. Try 3.8 or 3.9 for the time being.

**Comment by t3chman** [comment]
Can confirm that 3.10 was not working for me and had to revert to 3.8

**Comment by 0xLuca** [comment]
> Right, someone claimed this was incompatible before. Try 3.8 or 3.9 for the time being.

After downgrading to 3.8 the build succeeds without any problems and the server runs. 
Thank you for you the fast issue processing.

**Comment by gottfriedleibniz** [comment]
Was https://github.com/citizenfx/fivem/commit/db07118aa089244b2dfbfdfd0af4c03dd77e2300 related to this issue?
